src/database/database.ts

```diff
@@ -47,9 +53,16 @@ import {
 
 // Create the database class
 const sabr = new Sabr();
+sabr.directoryPath = `${
```

Comment:
would this work if the directory structure is as follow

root
  skillz
    gamer-deno
    db

Reply:
Yes, if you follow a few rules at starting this works.